### ISSUE TITLE
Search modal form is having white background

### ISSUE THREAD

**Issue by ikunali** [widget-/slack-black-theme#61]
Slack [added](https://slackhq.com/less-searching-more-doing-d249e59fa8c1) a _search_ form, which is complete white. Please add CSS to make it dark too

**Comment by caiceA** [comment]
@TG-KK i can give you css for the complete black theme i customise myslef solving some issues myself. if you want
<img width="1915" alt="screen shot 2018-07-28 at 00 04 20" src="https://user-images.githubusercontent.com/16766231/43348328-da55d322-91f9-11e8-8142-e8160f26262b.png">
<img width="962" alt="screen shot 2018-07-28 at 00 04 26" src="https://user-images.githubusercontent.com/16766231/43348329-da734f9c-91f9-11e8-8825-38425b268300.png">
<img width="1919" alt="screen shot 2018-07-28 at 00 04 35" src="https://user-images.githubusercontent.com/16766231/43348330-da91bf9a-91f9-11e8-8b77-b79951e428e8.png">

**Comment by ikunali** [comment]
@caiceA please do. Your theme looks amazing!

**Comment by philip41201** [comment]
i would love to have this theme too!

**Comment by caiceA** [comment]
@TG-KK @philip41201 sorry i was busy i will send you all the information and file today you can replace this with your file and BOOMMMM !!!

**Comment by ghost** [comment]
@caiceA I've been messing with the styles forever and can't get it quite right. Your screen shot looks fantastic! Do you mind sharing with me as well? Thanks!

**Comment by jonahx** [comment]
@caiceA +1

**Comment by b6b** [comment]
@caiceA Can you put it up in a gist or something so anyone can use it?

**Comment by CaptainStarPaw** [comment]
Great work @caiceA 

I'd love to use the same theme if you wouldn't mind sharing? Thank you so much.

**Comment by caiceA** [comment]
@TG-KK @philip41201 @aventak @b6b @captainstarpaw 
sorry for the delay I am finding the file I saved separately. please give me one day I will find again and will share with all you guys your slack will become so good like this. 


<img width="962" alt="screen shot 2018-08-01 at 13 41 54" src="https://user-images.githubusercontent.com/16766231/43519849-3af7396a-9591-11e8-9145-3012c72d4b19.png">
<img width="1920" alt="screen shot 2018-08-01 at 13 42 12" src="https://user-images.githubusercontent.com/16766231/43519850-3b12548e-9591-11e8-86cb-f726f7885f95.png">

**Comment by dotpyfe** [comment]
@caiceA would you mind also sending me this file? cheers

**Comment by mikejk8s** [comment]
@caiceA Please share this in a github repo, thanks!

**Comment by P05TPWND** [comment]
@caiceA 
Hi there, can you share this in the repo please?

**Comment by ikunali** [comment]
@caiceA Do we need to watch this new repo?
https://github.com/caiceA/Slack-in-Black

**Comment by caiceA** [comment]
@TG-KK  Yes, I created that Rep. for you and others. but couldn't load files I was away for vacations

**Comment by CaptainStarPaw** [comment]
@caiceA Are you trolling us? 26 days ago you said you needed a day to find the file. Now you're saying you went on vacation. Not trying to be a dick, but we don't need to hear the excuses. If you're gonna post the file, post it 👍

**Comment by caiceA** [comment]
@captainstarpaw First of all 
I don't need to be a dick when you already.
second of all if you ever lived in EU you may now August full month is Off in Spain only vacations
last. I don't owe you anything so get over yourself.

**Comment by CaptainStarPaw** [comment]
@caiceA Thanks for the condescending update!

**Comment by caiceA** [comment]
@TG-KK special thanks for your patience
@captainstarpaw  thank you for your comments. 

### https://github.com/caiceA/Slack-In-Black

Let me know if there is any problem thanks to everyone for patience 
Enjoy Everyone 
KC

**Comment by ikunali** [comment]
@caiceA Thanks so much! Anything free as well as awesome is worth waiting for! 🍭

**Comment by CaptainStarPaw** [comment]
@caiceA Gracias por la entrega. Por cierto, no quería ofender. Pensé que después de las excusas era una promesa falsa, pero estaba equivocado. Eres el mejor, gracias de nuevo 😋

**Comment by caiceA** [comment]
@aventak @Haaroon @zackhsi @jonahx @b6b @widget- @zackhsi @justync7 @dotpyfe @P05TMAN @philip41201 @ngriffin7a @mikejk8s  
Repository Added above 
Thanks

**Comment by CaptainStarPaw** [comment]
@caiceA Small but nice addition. You can also add a :hover and :focus state for #primary_file_button

`  #primary_file_button:hover, #primary_file_button:focus {
	  border: 2px solid #00a462 !important;
	  background: #00a462 !important;
  }`

The above example is keeping the green colour, and it simply lightens when hovering/interacting. I changed it to blue for my own personal use, but I think the transition makes it behave more naturally.

**Comment by caiceA** [comment]
@captainstarpaw thanks I added you're to the file plus made some cosmetic changes in the search box if you want to the latest file from repo or download from the link

**Comment by jrausch12** [comment]
@caiceA you are a wonderful human being. My eyes (and OCD) graciously thank you.

**Comment by caiceA** [comment]
@jrausch12  thank you !!
